ces_client/utils.py

```diff
@@ -32,15 +32,18 @@ def create_msg_from_ben(self, msg):
             return 'Oh.', answers
 
         state, closing_state = self.find_user_state(msg)
+
         if not closing_state:
             text = state.message.text
+            prepped_text = self.__class__.prep_text_for_botui(text)
```

Comment:
self.__class__ is not necessary. You can just call self.prep_text_forbotui(text)